src/uwtools/file.py

```diff
@@ -55,7 +56,7 @@ def _file_map(self) -> dict:
         for key in self._keys:
             nav.append(key)
             if key not in cfg:
-                raise UWConfigError("Config navigation %s failed" % " -> ".join(nav))
+                raise UWConfigError("Failed following YAML key(s): %s" % " -> ".join(nav))
```

Comment:
I ran into this error when testing, where my file dst/src block was under a top-level `files` key. The error read
```
Config navigation files failed
```
which I initially misinterpreted. The updated error would read
```
Failed following YAML key(s): files
```
which I hope is clearer. It definitely helped me, as I'd added `files` to my CLI invocation to access the block.